analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -187,11 +187,18 @@ abstract class PackageManager(
 
     /**
      * Return a tree of resolved dependencies (not necessarily declared dependencies, in case conflicts were resolved)
-     * for all [definitionFiles] which were found by searching the [analyzerRoot] directory.
+     * for all [definitionFiles] which were found by searching the [analyzerRoot] directory. By convention, the
+     * [definitionFiles] must be absolute.
      */
     open fun resolveDependencies(definitionFiles: List<File>): ResolutionResult {
         val result = mutableMapOf<File, ProjectAnalyzerResult>()
 
+        definitionFiles.forEach { definitionFile ->
```

Comment:
I'd move it even before the declaration of result.

Reply:
Done.